src/collective/cover/setuphandlers.py

```diff
@@ -1,8 +1,21 @@
 # -*- coding: utf-8 -*-
+from collective.cover.config import PROJECTNAME
+from plone.dexterity.interfaces import IDexterityFTI
 from Products.CMFPlone import interfaces as Plone
 from Products.CMFQuickInstallerTool import interfaces as QuickInstaller
+from zope.component import queryUtility
 from zope.interface import implements
 
+import logging
+import pkg_resources
+
+try:
+    pkg_resources.get_distribution('plone.app.stagingbehavior')
+except pkg_resources.DistributionNotFound:
+    IStagingSupport = None
+else:
+    from plone.app.stagingbehavior.interfaces import IStagingSupport
```

Comment:
The code between a try/except should be as small as possible. So I like it how it currently is. I don't think it's less readable.